src/viking.js

```diff
@@ -1,11 +1,90 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor(health,strength){
+      this.health = health;
+      this.strength = strength;
+    }
+    attack(){
+        return this.strength;
+      };
+    receiveDamage(damage){
+        this.health -= damage;
+      };
+    
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier{
+    constructor(name,health,strength){
+        super(health,strength)
+        this.name = name;
+    }
+    battleCry(){
+            return  "Odin Owns You All!";
+    };
+    receiveDamage (damage){
+            let pointsLife = this.health -= damage;
+            if( pointsLife > damage){
+                return `${this.name} has received ${damage} points of damage`;
+            }else{
```

Comment:
Oi Shania, aqui na verdade pointsLife no if, deverá ter a condicional se pointsLife for maior que zero,